src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj

```diff
@@ -1856,11 +1855,11 @@
 				CLANG_CXX_LIBRARY = "compiler-default";
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = TogglDesktop/TogglDesktop.entitlements;
-				CODE_SIGN_IDENTITY = "Mac Developer";
+				CODE_SIGN_IDENTITY = "-";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
-				DEVELOPMENT_TEAM = B227VTMZ94;
-				ENABLE_HARDENED_RUNTIME = YES;
```

Comment:
I suggest the we shouldn't disable the `ENABLE_HARDENED_RUNTIME` because it's a crucial step to submit for Apple Notarization. If we set NO, we couldn't submit.

Reply:
It turns out that if we remove code signing in `DEVELOPMENT_TEAM` and `CODE_SIGN_IDENTITY`, the app won't build until we disable `ENABLE_HARDENED_RUNTIME`. Thus, we have to find another way to disable the codesigning.